src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -341,8 +359,14 @@ public Bucket createS3Bucket(String name) throws IOException {
 
     private Bucket createS3Bucket(String name, boolean disableSessionToken) throws IOException {
         AmazonS3ClientBuilder builder = getAmazonS3ClientBuilderWithCredentials(disableSessionToken);
-        AmazonS3 client = builder.build();
-        return client.createBucket(name);
+        //Creating a new bucket cannot use transfer acceleration
+        AmazonS3 client = builder.withAccelerateModeEnabled(false).build();
```

Comment:
AFAICS, this is not useful, the bucket can be created just fine keeping the previous implementation.

Reply:
Correct, the bucket can be created while using accelerated mode. Unfortunately, for some reason, you can't set the accelerated mode on a bucket while your client is in accelerated mode. I've updated the comment to make that clearer